packages/addons/service/dispmanx_vnc/source/system.d/service.system.dispmanx_vnc.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=dispmanx_vnc
+After=kodi.service
+
+[Service]
+ExecStart=/bin/sh /storage/.kodi/addons/service.system.dispmanx_vnc/bin/dispmanx_vncserver-service
```

Comment:
Hi @MilhouseVH
What's in a name? :D
I guess I got used to end the name of my service wrappers with -service (see also inadyn, syncthing and tinc). By the way, I have seen such scripts ending with -start. So I presume it is a matter of convention.
Regards, @